specs/AppLifecycle/Restart/restartApi.markdown

```diff
@@ -64,135 +69,177 @@ There are 2 related Win32 APIs that focus on recovery prior to restart:
+        UInt32 ProcessId{ get; };
+    }
+
+    static runtimeclass ActivationRegistrationManager
+    {
+        static void RegisterForFileTypeActivation(String[] supportedFileTypes, String logo,
+            String displayName, String[] supportedVerbs, String exePath);
+        static void RegisterForProtocolActivation(String scheme, String logo, String displayName,
+            String exePath);
+        static void RegisterForStartupActivation(String taskId, String exePath);
+
+        static void UnregisterForFileTypeActivation(String[] fileTypes, String exePath);
+        static void UnregisterForProtocolActivation(String scheme, String exePath);
+        static void UnregisterForStartupActivation(String taskId);
+    };
+}
+```
+
+### Description
+
+Equivalent to CoreApplication.RequestRestartAsync, except that it is *not* async.
+- If the restart fails, but the user subsequently launches the app manually, the app will launch normally and no restart arguments will be passed.
+- If the app has any in-process background tasks running when it calls this API, those tasks will be cancelled in the normal way. Out-of-process background tasks will not be affected.
+
+The restarted instance will be activated with Windows.ApplicationModel.Activation.LaunchActivatedEventArgs.
+- When the app is restarted, LaunchActivatedEventArgs.PreviousExecutionState will have the value Terminated so that the app can distinguish between a resume and a restart.
+- The elevation state of the app is also maintained in the new restarted process.
+
+If the request to restart is successful, the application will be restarted immediately with no return value. The user does not have to do anything else/handle a return value in any way.
 
 If the request fails due to a nonfatal reason, a failure reason is returned for why the restart didn’t successfully complete. This method leverages an existing enum (AppRestartFailureReason) which has the following relevant return fields.
-<ul>
-<li> InvalidUser: Could not restart for the specified user.</li>
-<li> RestartPending: A restart is already in progress.</li>
-<li> Other: Unspecified failure.</li>
-</ul>
-</p></td>
-</tr>
-</tbody>
-</table>
-
-When an application calls RequestRestartNow, the mechanism for restart will be as follows:
-1. Application calls RequestRestartNow API.
-2. The API will CreateProcess with the Agent (a helper EXE in the framework package containing the API) and will then wait for an an acknowledgment (ACK) from the Agent.
-3. The Agent will get the executable path of the application and signal an ACK to the API. The Agent will then then terminate the application and then CreateProcess with the application (restarted).
-
-## Examples and Scenarios
+- InvalidUser: Could not restart for the specified user.
+- RestartPending: A restart is already in progress.
+- Other: Unspecified failure.
+
+### Mechanism
+
+When an application calls Restart, the mechanism for restart will be as follows:
```

Comment:
Markdown nit: Put a blank line before & after number/bullet lists.

Reply:
Done for all lists!